### PR TITLE
Caesar cipher plugin added

### PR DESCRIPTION
**Pull request by shotanozadze**
This new plugin converts text into encrypted one (caesar cipher)

### CODE REVIEW

File: jarviscli/plugins/caesar_cipher.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    jarvis.say(converted, Fore.YELLOW)
+
+
+def cipher_to_plain(jarvis):
+    user_input = get_user_input(jarvis)
+    converted = ""
+
+    for i in user_input:
+        if i.isalpha():
```
Comment (pnhofmann) [inline jarviscli/plugins/caesar_cipher.py:60]: Good work!

Only noticed, you did not consider Unicode. Something like for example our German: ä, ü, ö, ß ;). As a suggestion, you could instead isalpha do a check similar to:
https://stackoverflow.com/questions/196345/how-to-check-if-a-string-in-python-is-in-ascii#answer-196392

Comment (shotanozadze) [reply jarviscli/plugins/caesar_cipher.py:60]: Ok, I'll fix it

**Comment by shotanozadze** [comment]
Done! @pnhofmann

**Comment by pnhofmann** [comment]
Thanks!

**Pull request merged**